crates/accelerate/src/permutation.rs

```diff
@@ -111,10 +116,31 @@ fn _get_ordered_swap(py: Python, permutation_in: PyArrayLike1<i64>) -> PyResult<
     Ok(get_ordered_swap(&view).to_object(py))
 }
 
+#[pyfunction]
+#[pyo3(signature = (pattern))]
+fn _synth_permutation_basic(py: Python, pattern: PyArrayLike1<i64>) -> PyResult<CircuitData> {
+    let view = pattern.as_array();
+    let num_qubits = view.len();
+    Ok(CircuitData::from_standard_gates(
+        py,
+        num_qubits as u32,
+        get_ordered_swap(&view).iter().map(|(i, j)| {
+            (
+                StandardGate::SwapGate,
+                smallvec![],
+                smallvec![Qubit(*i as u32), Qubit(*j as u32)],
+            )
+        }),
+        Param::Float(0.0),
+    )
+    .expect("Something went wrong in Qiskit's Python realm"))
```

Comment:
I'm going to enjoy when we have a user complaining about an uncatchable `PanicException` complaining about the python realm when calling `Qubit()` fails unexpectedly. :p

Reply:
Yup that'll inevitably happen at some point 😅 is there a better way of handling it right now? I was mimicking how it's done in other place so far 😬